### PR TITLE
chore: rename slotsPerNode to maxSlotsPerPod [DET-4066]

### PR DESCRIPTION
**Pull request by aaron276h**

### CODE REVIEW

File: docs/how-to/installation/kubernetes.txt [hunk 1]
```diff
@@ -64,11 +64,11 @@ Number of GPUs Per Node
...
 
 Users are required to specify the number of GPUs on each node (for GPU-enabled nodes only). This is done
-by setting ``slotsPerNode`` in ``values.yaml``. Determined uses this information when scheduling multi-GPU tasks.
-Each multi-GPU (distributed training) task will be scheduled as a set of ``slotsPerTask / slotsPerNode`` separate pods,
-with each pod assigned to ``slotsPerNode`` GPUs. Tasks with sizes that are not divisible by ``slotsPerNode`` are never scheduled.
-If you have a cluster of different size nodes set ``slotsPerNode`` to the smallest common denominator. For example, if
-you have nodes with 4 GPUs and other nodes with 8 GPUs, set ``slotsPerNode`` to 4 so that all distributed experiments
+by setting ``maxSlotsPerPod`` in ``values.yaml``. Determined uses this information when scheduling multi-GPU tasks.
+Each multi-GPU (distributed training) task will be scheduled as a set of ``slotsPerTask / maxSlotsPerPod`` separate pods,
+with each pod assigned up to ``maxSlotsPerPod`` GPUs. Tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.
```
Comment (neilconway) [inline docs/how-to/installation/kubernetes.txt:69]: > Tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.

Maybe not worth making this more complicated than it already is, but would that incorrectly imply that tasks whose size is smaller than `maxSlotsPerPod` won't be scheduled?

File: master/go.sum [hunk 2]
```diff
@@ -218,6 +218,7 @@ github.com/envoyproxy/go-control-plane v0.9.0/go.mod h1:YTl/9mNaCwkRvm6d1a2C3ymF
 github.com/envoyproxy/go-control-plane v0.9.1-0.20191026205805-5f8ba28d4473/go.mod h1:YTl/9mNaCwkRvm6d1a2C3ymFceY/DCBVvsKhRF0iEA4=
 github.com/envoyproxy/go-control-plane v0.9.4/go.mod h1:6rpuAdCZL397s3pYoYcLgu1mIlRU8Am5FuJP05cCM98=
 github.com/envoyproxy/protoc-gen-validate v0.1.0/go.mod h1:iSmxcyjqTsJpI2R4NaDN7+kN2VEUnK/pcBlmesArF7c=
+github.com/evanphx/json-patch v4.2.0+incompatible h1:fUDGZCv/7iAN7u0puUVhvKCcsR6vRfwrJatElLBEf0I=
```
Comment (neilconway) [inline master/go.sum:221]: Are these changes to `go.sum` intentional?

Comment (aaron276h) [reply master/go.sum:221]: Ah this shouldn't be in here.

File: docs/how-to/installation/kubernetes.txt (hunk 1 above)
Comment (aaron276h) [reply docs/how-to/installation/kubernetes.txt:69]: How about:

```
Distributed tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.
```

Comment (neilconway) [reply docs/how-to/installation/kubernetes.txt:69]: sgtm

**Pull request merged**